validator/nodejs/postinstall-windows.js

```diff
@@ -20,6 +20,18 @@
 var path = require('path');
 var fs = require('fs');
 
+fs.unlink("postinstall.DELETEME", (err) => {
+	if (err) {
+		console./*OK*/log("Could not delete postinstall.DELETEME. Please delete it.");
+	}
+});
+
+fs.exists('', function(exists) {
```

Comment:
I think I should stop working on a saturday :smile: I'll do the changes you requested